src/components/challengeWidget/challengeWidget.js

```diff
@@ -0,0 +1,40 @@
+//import logo from './logo.svg';
+import React from 'react';
+//import fontawesome from '.Components/fontawesome';
+import './challengeWidget.css';
+
+function App() {
+  return (
+    <div className="App">
+      <a href="#"><h1>Read more in 2021</h1></a>
```

Comment:
We need a few component props here:

- `title` which goes within the `<h1>` (which by the way should probably be a `<h2>`)
- `to` which will be used in the `<Link>`